modules/management-console-agent/pom.xml

```diff
@@ -51,19 +51,19 @@
         <dependency>
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-client</artifactId>
-            <version>${jetty.version}</version>
+            <version>9.4.22.v20191022</version>
```

Comment:
Why you changed version here? But not in parent/pom.xml ?

Reply:
It is discussed with @smoldachev in parent/pom.xml 9.4.18 version of jetty is returned.
9.4.22 version is used only for management-console-agent project.

Reply:
Version reverted.